crates/runix/src/flake_ref.rs

```diff
@@ -148,8 +148,8 @@ impl ToFlakeRef {
                 unpack: _,
                 nar_hash: _,
             } => todo!(),
-            ToFlakeRef::Indirect(IndirectFlake { id: _ }) => {
-                Url::parse("flake:{id}").expect("Failed to create indirect reference")
+            ToFlakeRef::Indirect(IndirectFlake { id }) => {
+                Url::parse(&format!("flake:{0}", id)).expect("Failed to create indirect reference")
```

Comment:
```suggestion
                Url::parse(&format!("flake:{id}")).expect("Failed to create indirect reference")
```
would this work instead? I imagine that was the original intention

Reply:
Yes that should work.